book/src/lint_configuration.md

```diff
@@ -540,4 +541,12 @@ if no suggestion can be made.
 * [indexing_slicing](https://rust-lang.github.io/rust-clippy/master/index.html#indexing_slicing)
 
 
+### missing-docs-in-crate-items
+Whether to **only** check for missing documentation in `pub(crate)` items.
```

Comment:
Can this be updated in the same way as https://github.com/rust-lang/rust-clippy/pull/10303/files#diff-1d884fd439e15a9e47e1135cdc0ab5ee896a876b9f4c256ecce4be6c69a8831dR459?